lib/wikitrivia/game.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Wikitrivia.Game do
+  def create_game do
+    game_id = Ecto.UUID.generate
+    {:ok, _} = Agent.start_link(fn -> default_state end, name: agent_name_by_game_id(game_id))
+    game_id
+  end
+
+  def get_game_state(game_id) do
+    Agent.get(agent_name_by_game_id(game_id), fn s -> s end)
+  end
+
+  def add_player(game_id, player_id, player_name) do
```

Comment:
@alexcavalli I like your idea of having the player name as the ID. That would also help ensure names are unique.

Reply:
@denheck So, then, an initial structure maybe like this?
```elixir
%{
  players: ["player_1_name", etc] # MapSet or List or something, depending on if we want to retain order
  scores: %{
    "player_1_name" => 10,
    etc.
  } 
}
```